src/rabbitmq/rabbitmq.module.ts

```diff
@@ -0,0 +1,21 @@
+import { Module } from '@nestjs/common';
+import { RabbitMQModule } from '@golevelup/nestjs-rabbitmq';
+import { RabbitmqService } from './rabbitmq.service';
+
+@Module({
+  imports: [
+    RabbitMQModule.forRoot(RabbitMQModule, {
+      exchanges: [
+        {
+          name: 'ticktalk_exchange',
+          type: 'direct',
+        },
+      ],
+      uri: process.env.RABBITMQ_URI,
+      connectionInitOptions: { wait: false },
+    }),
+  ],
```

Comment:
I suggest we get rid of `dotenv` and use the `ConfigService` all over the application to get environment variables. Let's get the uri from the `.env` using the `ConfigService`
```suggestion
  imports: [
    ConfigModule,
    RabbitMQModule.forRootAsync(RabbitMQModule, {
      inject: [ConfigService],
      useFactory: (configService: ConfigService) => ({
        exchanges: [
          {
            name: 'ticktalk_exchange',
            type: 'direct',
          },
        ],
        uri: configService.get<string>('RABBITMQ_URI'), // Access environment variable
        connectionInitOptions: { wait: false },
      }),
    }),
  ],
```